bms/bms_app/views.py

```diff
@@ -73,13 +73,14 @@ def website_home(request):
     return render(request, 'registration/website_home.html')
 
 @login_required(login_url='/login/')
-def create_product(request):
+def create_project(request):
 
     if request.method == 'GET':
         users = User.objects.filter()
         roles = UserRole.objects.filter()
-        prod_types = Product_type.objects.filter()
-        return render(request, 'registration/create_product.html', 
+        prod_types = ProjectType.objects.filter()
```

Comment:
why you have used filter() method to get the project type, user, roles, it should be use to filter data.